v2.0/orchestrate-cockroachdb-with-kubernetes-insecure.md

```diff
@@ -70,7 +70,22 @@ instance | A physical or virtual machine. In this tutorial, you'll create GCE or
     cockroachdb-3   1/1       Running   0          46s
     ~~~
 
-## Step 9. Stop the cluster
+## Step 9. Upgrade the cluster
+
+{% include orchestration/kubernetes-upgrade-cluster.md %}
+
+4. If this was an upgrade between minor or major versions (e.g. between v1.0.x and v1.1.y or between v1.1.y and v2.0.z), then you'll want to [finalize the upgrade](upgrade-cockroach-version.html#finalize-the-upgrade) if you're happy with the new version. Assuming you upgraded to the v1.1 minor version, you'd run:
```

Comment:
nit: `e.g.` > `e.g.,`

Reply:
Also, since this is the 2.0 version of the page, should we say `Assuming you upgraded to the v2.0 minor version...`?

Reply:
Good catch, done.